src/common/config/config.json

```diff
@@ -1,17 +1,13 @@
 {
-    "theme": {
-        "primaryColor": "#67D7ED",
-        "secondaryColor": "#03dac4",
-        "tertiaryColor": "white",
-        "linkColor": "#0057B2"
-    },
-    "appLogoUrl": "/tonomy-logo48.png",
+    "themeFile": "theme.pangea.css",
+    "appLogoUrl": "/favicon.png",
```

Comment:
app logo should also be white labelable, depending on the environment.

Reply:
it is fetching from public directory